### PR TITLE
Chore // Bumped up test version

### PR DESCRIPTION
**Pull request by stuartjohnpage**
### Overview

I forgot to bump up the test version in #446 - this should sort that out

### CODE REVIEW

**Review by grossvogel — APPROVED** [review]

File: lib/slax_web/websockets/poker.ex [hunk 1]
```diff
@@ -89,7 +89,7 @@ defmodule SlaxWeb.Poker do
         estimates =
           round.estimates
           |> Enum.sort_by(& &1.value)
-          |> Enum.map(&"#{&1.user} (#{&1.value})#{if &1.reason, do: ': #{&1.reason}'}")
+          |> Enum.map(&"#{&1.user} (#{&1.value})#{if &1.reason, do: ~c": #{&1.reason}"}")
```
Comment (grossvogel) [inline lib/slax_web/websockets/poker.ex:92]: Anybody know why we're using charlists here?

Comment (stuartjohnpage) [reply lib/slax_web/websockets/poker.ex:92]: That's really funky

Comment (stuartjohnpage) [reply lib/slax_web/websockets/poker.ex:92]: It also shows up as funky from the syntax highlight in my vscode now; but it does still work

Comment (stuartjohnpage) [reply lib/slax_web/websockets/poker.ex:92]: wait that's just being the `" ` and the `:` are so close together

**Pull request merged**